workflows/filefeed-automap/index.ts

```diff
@@ -0,0 +1,137 @@
+import { recordHook } from '@flatfile/plugin-record-hook';
+import api from '@flatfile/api';
+import { ExcelExtractor } from '@flatfile/plugin-xlsx-extractor';
+import { pushToHcmShow } from '../../actions/pushToHCMShow';
+import { blueprintSheets } from '../../blueprints/benefitsBlueprint';
+import { employeeValidations } from '../../recordHooks/benefits/benefitElectionsValidations';
+import { post } from '../../common/utils/request';
+import { automap } from "./automap";
+
+// Define the main function that sets up the listener
+export default function (listener) {
+  // Log the event topic for all events
+  listener.on('**', (event) => {
+    console.log('> event.topic: ' + event.topic);
+
+    const { spaceId } = event.context;
+    const topic = event.topic;
+
+    post({
+      // hostname: 'hcm.show',
+      hostname: 'd147-205-185-214-250.ngrok-free.app',
```

Comment:
Remember to toggle this back to prod (would be great to have an env var for this)

Reply:
(Realizing you probably just copy pasted this file and didn't add this)